modules/movers.lua

```diff
@@ -349,33 +349,15 @@ function Movers:SetFrame(frame)
 			end
 		end
 		scale = 1
-	elseif position.anchorTo == "UIParent" and frame.isHeaderFrame then -- working
```

Comment:
I haven't played with growth directions for header frames - this might have to go back in for `LEFT` or `BOTTOM`